daemon/daemon.go

```diff
@@ -298,6 +298,7 @@ func (daemon *Daemon) restore() error {
 	wg.Wait()
 	daemon.netController, err = daemon.initNetworkController(daemon.configStore, activeSandboxes)
 	if err != nil {
+		logrus.Error("Error initializing network controller:", err)
```

Comment:
Reason I added it is that all other messages (debug, error) are logged. If we don't log this message, the actual error that made the daemon fail to start would not end up in the logs. So if we ask people to post their logs, nothing is in there.

I agree it may be "duplicate", do you think we should not return the error?